mkt/reviewers/utils.py

```diff
@@ -419,6 +429,14 @@ def get_actions(self):
                              u'The message won\'t be visible to the '
                              u'author(s), and no notification will be sent '
                              u'them.')}
+        manual_rereview = {
+            'method': self.handler.process_manual_rereview,
+            'label': _lazy(u'Request Re-review'),
+            'minimal': True,
+            'details': _lazy(u'Add this app to the re-review queue. Any '
+                             u'comments here won''t be visible to the '
```

Comment:
Yeah it works too, it's just more annoying in translations IMHO, that's why we usually use double quotes when there are `'` in the string